tests/deployment/test_client_server.py

```diff
@@ -95,10 +98,21 @@ def test_client_server_sklearn(
     # Compile
     extra_params = {"global_p_error": 1 / 100_000}
 
+    if get_model_name(model_class) == "KNeighborsClassifier":
+
+        default_configuration = Configuration(
+            dump_artifacts_on_unexpected_failures=False,
+            enable_unsafe_features=True,
+            use_insecure_key_cache=True,
+            insecure_key_cache_location="ConcreteNumpyKeyCache",
+            parameter_selection_strategy=fhe.ParameterSelectionStrategy.MONO,
+            single_precision=True,
+        )
+
     # Running the simulation using a model that is not compiled should not be possible
     with pytest.raises(AttributeError, match=".* model is not compiled.*"):
         client_server_simulation(x_train, x_test, model, default_configuration)
-
+    # With n_bits = 3, KNN is not compilable
```

Comment:
but don't we want to leave the n_bits setting as it was for the other models ? Is it doable ? 

if it doesn't matter that we reduce n_bits to 2 for all modesl and it's faster this way (are we doing FHE computation herE), then we can set it to 2 all the time and gain some CI time

Reply:
Yes, we are doing FHE computation here, but it doesn't impact the other models and we gain mere seconds in the CI.
So, I thought we can keep 2 for all models.